lib/utils/schematic/getAllDimensionsForSchematicBox.ts

```diff
@@ -183,21 +183,21 @@ export const getAllDimensionsForSchematicBox = (
   currentDistanceFromEdge = 0
   // moving rightward from the left-bottom corner
   for (let sideIndex = 0; sideIndex < sidePinCounts.bottomSize; sideIndex++) {
-    const pinNumber =
+    const pinNumberOrAlias =
       params.schPortArrangement &&
       isExplicitPinMappingArrangement(params.schPortArrangement)
         ? params.schPortArrangement.bottomSide?.pins[sideIndex]!
         : truePinIndex + 1
     const pinStyle =
-      params.schPinStyle?.[`pin${pinNumber}`] ?? params.schPinStyle?.[pinNumber]
+      params.schPinStyle?.[`pin${pinNumberOrAlias}`] ?? params.schPinStyle?.[pinNumberOrAlias]
```

Comment:
This change is wrong, it ALWAYS be `pin${pinNumber}`

```suggestion
      params.schPinStyle?.[`pin${pinNumberOrAlias}`] ?? params.schPinStyle?.[pinNumberOrAlias]
```

Reply:
if there is an alias, we can check for `params.schPinStyle[alias]`